micom/workflows/build.py

```diff
@@ -147,14 +148,22 @@ def build_database(
         Must contain the columns "file", "kingdom", "phylum", "class",
         "order", "family", "genus", and "species". May contain additional
         columns.
+
     out_path : str
-        The directory where the joined models will be written.
+        The directory or zip file where the joined models will be written.
+
     threads : int >=1
         The number of parallel workers to use when building models. As a
         rule of thumb you will need around 1GB of RAM for each thread.
-    compress : bool
-        Whether to compress the output. Default is True if out_path ends with
-        ".zip" otherwise no.
+
+    compress : False (default, no compression), "zlib", "bz2", or "lzma"
```

Comment:
probably should be 

```
compress : str (default None)
    Compression method to use. Must be one of "zlib", "bz2", or "lzma".
    This parameter is ignored if out_path does not end with ".zip".
```

None is probably the better default since False would imply the existence of True as well.